### ISSUE TITLE
Bug: The configure option of dash is not POSIX compatible.

### ISSUE THREAD

**Issue by carbongreat13** [void-linux/void-packages#35457]
### Expected behavior
dash is a "POSIX-compliant Unix shell" in the description. However, the current build options are not POSIX-compatible. You need to include `--with-libedit` as a build option.

See: https://bugs.archlinux.org/task/36346

**Comment by leahneukirchen** [comment]
False, the command line editing feature is part of the "User Portability Utilities option", a noninteractive sh can very well be POSIX-compatible. (Also see `posh` for example.)

I think we have enough better interactive shells packaged, so imo WONTFIX.

**Comment by erwin** [comment]
I realize this issue is closed, but I would ask you to reconsider including `--with-libedit` with `dash` on usability grounds.

1, This week I did a full re-install of void, and while making some changes to `/etc/fstab` ended up getting dropped into single user mode a few times... Sure you can start another shell, but it's fairly painful to use dash without even basic command line editing.

2, Also, Docker is hardcoded to use `/bin/sh` for build files... So when using dash to troubleshoot individual `RUN` commands from a Dockerfile, would be nice to have some basic command line editing.

3, Those are two examples that I ran into this week. Without line editing, it's just so unfriendly... Like a trip back to a pre-Linux terminal.

4. It's only 219K (though dash is only 123K), and I don't think dash is statically linked anyway.

Bottom line, dash exists there because sometimes we do actually have to use it, and when we do have to, it's usually in a troublesome time (like single user mode).

Thanks for taking the time to read this.
